paparazzi/paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -564,6 +564,14 @@ class Paparazzi @JvmOverloads constructor(
       .apply { isAccessible = true }
       .get(dispatcher) as ArrayDeque<*>
     toRunTrampolined.clear()
+    dispatcher.javaClass
+      .getDeclaredField("scheduledTrampolineDispatch")
+      .apply { isAccessible = true }
+      .set(dispatcher, false)
+    dispatcher.javaClass
+      .getDeclaredField("scheduledFrameDispatch")
+      .apply { isAccessible = true }
+      .set(dispatcher, false)
```

Comment:
One day the Compose Runtime will probably fix these memory leaks.  At the point, will we still need this?  

If so, maybe add a comment for future contributors to remember not to remove this?

Reply:
I'd imagine if those leaks were fixed that this would go as well since us clearing the callback lists without reseting these values is whats getting us into this situation in the first place. They would have to handle resetting them as part of that larger fix else they would run into the situation we are today.

The included plugin test should catch any regression's if/when we remove this function, but can leave a comment to double check.